nss-day-cohort-45/nutyshell-talking-with-a-full-mouth-1
# Added userId to be saved in the api for each new saved event.

# Description
Added code so that the userId is to each saved event object and so a dialog box pops up when you click Show Weather.

## Type of change
- [✔️] New feature (non-breaking change which adds functionality)

# Testing Instructions
1. Save a new event and check your local database to ensure the new event has been saved with a userId.
2. Click on Show Weather button. Dialog box should open and you should be able to close it.

# Checklist:
- [✔️] My code follows the style guidelines of this project
- [✔️] I have performed a self-review of my own code
- [✔️] I have commented my code, particularly in hard-to-understand areas
- [✔️] My changes generate no new warnings or errors
- [✔️] I have added test instructions that prove my fix is effective or that my feature works
